src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,13 +3,37 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private Object[] keys;
+    private Object[] values;
+    private int index;
+
+    public StorageImpl() {
+        keys = new Object[ARRAY_SIZE];
+        values = new Object[ARRAY_SIZE];
+        index = 0;
+    }
 
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < index; i++) {
+            if (keys[i] != null && keys[i].equals(key)) {
```

Comment:
```suggestion
            if (keys[i] == key || keys[i] != null && keys[i].equals(key)) {
```
Let's accept one value by null key;